src/library/dlgtrackinfo.h

```diff
@@ -50,13 +50,7 @@ class DlgTrackInfo : public QDialog, public Ui::DlgTrackInfo {
     void slotCancel();
 
     void trackUpdated();
-
-    void slotBpmDouble();
-    void slotBpmHalve();
-    void slotBpmTwoThirds();
-    void slotBpmThreeFourth();
-    void slotBpmFourThirds();
-    void slotBpmThreeHalves();
+    void slotScaleBpm(const mixxx::Beats::BPMScale scale);
```

Comment:
The `const` keyword is discouraged when passing parameters by value

Reply:
ok, removed